### PR TITLE
Standardize Bulk Action Buttons

### PR DESCRIPTION
**Pull request by crhallberg**
Currently given `.action-toolbar` and `.toolbar-btn` classes which style them similar to `.btn-link`.

TODO
- [x] Fix issue: record toolbar has inconsistent alignment/formatting on "add to cart" button, and bold looks too heavy.
- [x] Run lessToSass before merging
- [x] Make sure full test suite passes

### CODE REVIEW

File: themes/bootstrap3/templates/RecordDriver/DefaultRecord/toolbar.phtml [hunk 1]
```diff
@@ -4,38 +4,40 @@
...
-    <li><a class="cite-record" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'Cite')?>" rel="nofollow"><i class="fa fa-asterisk" aria-hidden="true"></i> <?=$this->transEsc('Cite this')?></a></li>
-  <?php endif; ?>
-  <?php if ($this->accountCapabilities()->getSmsSetting() !== 'disabled'): ?>
-    <li><a class="sms-record" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'SMS')?>" rel="nofollow"><i class="fa fa-mobile" aria-hidden="true"></i> <?=$this->transEsc('Text this')?></a></li>
-  <?php endif; ?>
-  <li><a class="mail-record" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'Email')?>" rel="nofollow"><i class="fa fa-envelope" aria-hidden="true"></i> <?=$this->transEsc('Email this')?></a></li>
+<nav class="record-nav">
+  <ul class="action-toolbar">
+    <?php if (count($this->driver->getCitationFormats()) > 0): ?>
+      <li><a class="toolbar-btn" data-lightbox href="<?=$this->recordLink()->getActionUrl($this->driver, 'Cite')?>" rel="nofollow"><i class="fa fa-asterisk" aria-hidden="true"></i> <?=$this->transEsc('Cite this')?></a></li>
```
Comment (demiankatz) [inline themes/bootstrap3/templates/RecordDriver/DefaultRecord/toolbar.phtml]: Do we lose anything important by eliminating these more specific class names here? If we definitely do want to eliminate these, is there orphaned LESS somewhere that needs to be cleaned up?

File: themes/bootstrap3/templates/search/bulk-action-buttons.phtml [hunk 2]
```diff
@@ -1,26 +1,26 @@
 <?php if (isset($this->showCheckboxes) && $this->showCheckboxes): ?>
-  <div class="bulkActionButtons hidden-print">
+  <nav class="bulkActionButtons">
```
Comment (demiankatz) [inline themes/bootstrap3/templates/search/bulk-action-buttons.phtml]: Is hidden-print being removed intentionally?

File: themes/bootstrap3/templates/RecordDriver/DefaultRecord/toolbar.phtml (hunk 1 above)
Comment (crhallberg) [reply themes/bootstrap3/templates/RecordDriver/DefaultRecord/toolbar.phtml]: No, I looked for JS hooks and LESS styles and came up empty.

File: themes/bootstrap3/templates/search/bulk-action-buttons.phtml (hunk 2 above)
Comment (crhallberg) [reply themes/bootstrap3/templates/search/bulk-action-buttons.phtml]: Yes, I meant to tie it into the LESS but missed this step. Thank you!

Comment (crhallberg) [reply themes/bootstrap3/templates/search/bulk-action-buttons.phtml]: Nevermind, I already added it here: https://github.com/crhallberg/vufind/blob/bulk-normalize/themes/bootstrap3/less/components/search.less#L57

Comment (demiankatz) [reply themes/bootstrap3/templates/search/bulk-action-buttons.phtml]: Aha, thanks for clarifying that. At this point, what are the circumstances under which we would use .hidden-print in the template vs. a targeted `display:none` in the LESS? Just wondering if we have any inconsistencies that should be normalized....

**Comment by demiankatz** [comment]
@crhallberg, I think we're nearly there... but when we respond down to a smaller size, I think the buttons need a little vertical margin:
image

**Comment by demiankatz** [comment]
@crhallberg, just when you thought it was safe... one more glitch! The hover color for the cart button differs from all the other buttons. It's most obvious in bootprint3, but it's an issue in all three themes:
image
vs.
image
Can you fix? Thanks!

**Comment by crhallberg** [comment]
In the discussion of moving all icons to CSS, we have a few options. 

1. We can give each button a unique class: `.cart-bulk-save, .search-bulk-save`.
1. We can give each button a role class: `.btn-type-save`.
1. We can give each button an icon describer class `.btn-icon-star`.

These are all equally easy to accomplish, it's just a matter of what we think is best.

**Comment by demiankatz** [comment]
@crhallberg, my vote is for option 2 -- I think this strikes the right balance between semantic meaning, consistency and flexibility. Buttons that serve the same function can be styled in a single place, but the class name doesn't pre-determine the nature of that styling.

**Comment by demiankatz** [comment]
Victory at last! Thanks, @crhallberg!

**Pull request merged**